applications/FluidDynamicsApplication/symbolic_generation/navier_stokes/generate_navier_stokes_element.py

```diff
@@ -26,7 +26,7 @@
 dim_to_compute = "Both"             # Spatial dimensions to compute. Options:  "2D","3D","Both"
 divide_by_rho = True                # Divide the mass conservation equation by rho
 ASGS_stabilization = True           # Consider ASGS stabilization terms
-formulation = "NavierStokes"        # Element type. Options: "NavierStokes", "Stokes"
+formulation = "Stokes"        # Element type. Options: "NavierStokes", "Stokes"
```

Comment:
```suggestion
formulation = "NavierStokes"        # Element type. Options: "NavierStokes", "Stokes"
```
(just to keep the current default behavior)

Reply:
done.